iptables/iptables.go

```diff
@@ -40,7 +41,10 @@ func (e *Error) Error() string {
 }
 
 type IPTables struct {
-	path string
+	sync.Mutex
```

Comment:
Convention is generally that the Mutex is listed before the members it protects - but in this case it's not used for any of the below - could you move it to the end of the struct instead?